azurelinuxagent/common/telemetryevent.py

```diff
@@ -99,16 +99,12 @@ def is_extension_event(self):
         # "WALinuxAgent", it is an extension event.
         for param in self.parameters:
             if param.name == GuestAgentExtensionEventsSchema.Name:
-                return param.value != AGENT_NAME
+                if param.value != AGENT_NAME:
```

Comment:
wow. thanks for fixing this

Reply:
ohh lol I think I added something here regarding that priority queue initially but removed it. Let me revert this change

Reply:
Done